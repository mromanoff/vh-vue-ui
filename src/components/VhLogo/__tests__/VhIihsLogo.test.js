import { shallowMount } from '@vue/test-utils';
import VhIihsLogo from '../VhIihsLogo.vue';

describe('VhIihsLogo.vue', () => {
  let wrapper;

  it('renders default VhIihsLogo', () => {
    wrapper = shallowMount(VhIihsLogo);

    expect(wrapper).toMatchSnapshot();
  });
});
