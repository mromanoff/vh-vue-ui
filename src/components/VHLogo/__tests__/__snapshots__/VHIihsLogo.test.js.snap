// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VHIihsLogo.vue renders default VHContainer 1`] = `
<svg version="1.1" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" viewBox="0 0 750 374.76" xml:space="preserve" role="presentation" class="IIHSLogo">
  <path d="M335.13 219.2h57.58v148.49h-57.58zM335.13 7.07h57.58V139.4h-57.58z" class="path1"></path>
  <path d="M0 7.07h93.94v360.62H0zM116.66 7.07h93.94v360.62h-93.94zM241.19 7.07h93.94V139.4h57.57V7.07h93.94v360.62H392.7V219.2h-57.57v148.49h-93.94zM652.52 111.11v-8.58c0-21.22-8.58-38.89-26.77-38.89-20.2 0-29.29 15.66-29.29 31.82 0 71.21 153.54 36.36 153.54 168.18 0 76.77-45 111.12-127.78 111.12C544.44 374.76 501 348 501 273.24v-12.63h90.92v8.59c0 30.81 12.62 41.92 30.8 41.92 19.2 0 30.31-15.15 30.31-34.85 0-71.21-147.48-35.86-147.48-163.64C505.55 39.9 544.44 0 623.73 0 705.55 0 740.4 33.84 740.4 111.11z" class="path2"></path>
</svg>
`;

exports[`VHIihsLogo.vue renders default VHIihsLogo 1`] = `
<svg version="1.1" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" viewBox="0 0 750 374.76" xml:space="preserve" role="presentation" class="IIHSLogo">
  <path d="M335.13 219.2h57.58v148.49h-57.58zM335.13 7.07h57.58V139.4h-57.58z" class="path1"></path>
  <path d="M0 7.07h93.94v360.62H0zM116.66 7.07h93.94v360.62h-93.94zM241.19 7.07h93.94V139.4h57.57V7.07h93.94v360.62H392.7V219.2h-57.57v148.49h-93.94zM652.52 111.11v-8.58c0-21.22-8.58-38.89-26.77-38.89-20.2 0-29.29 15.66-29.29 31.82 0 71.21 153.54 36.36 153.54 168.18 0 76.77-45 111.12-127.78 111.12C544.44 374.76 501 348 501 273.24v-12.63h90.92v8.59c0 30.81 12.62 41.92 30.8 41.92 19.2 0 30.31-15.15 30.31-34.85 0-71.21-147.48-35.86-147.48-163.64C505.55 39.9 544.44 0 623.73 0 705.55 0 740.4 33.84 740.4 111.11z" class="path2"></path>
</svg>
`;
